avocado/utils/build.py

```diff
@@ -41,6 +42,15 @@ def make(path, make='make', env=None, extra_args='', ignore_status=False, allow_
     cwd = os.getcwd()
     os.chdir(path)
     cmd = make
+
+    # Set default number of jobs as ncpus + 1
+    jobs = str(multiprocessing.cpu_count() + 1)
+    if "-j" not in os.environ.get("MAKEFLAGS", ""):
+        if not env:
+            env = {"MAKEFLAGS": "-j%s" % jobs }
```

Comment:
there is an extra space after the `jobs`. While on it (I know it's really a minor detail) would you please move the `jobs to inside the first `if` branch? Also the `str` conversion is not necessary as python will convert it during `%s`.